### PR TITLE
feat: send flow

### PR DESCRIPTION
**Pull request by tiankii**
This pull request introduces several improvements and new tests for phone number handling and string normalization utilities, as well as expands the test coverage of the `SendBitcoinDestinationScreen`. The most significant changes are the addition of new unit tests for phone validation logic, enhancements to the test suite for destination parsing, and the export of the `phone-input` component.

**Test Coverage Improvements**

* Added comprehensive unit tests for phone number validation and parsing in `__tests__/utils/phone.spec.ts`, covering international formats, country codes, and edge cases.
* Expanded test cases for string normalization via the new `normalizeString` utility in `__tests__/utils/helper.spec.ts`, including handling whitespace, casing, and empty input.

**SendBitcoinDestinationScreen Test Enhancements**

* Refactored and extended the test suite in `__tests__/screens/send-destination.spec.tsx` to cover more scenarios, including phone number validation, username existence checks, clipboard input, and input focus switching.
* Improved mocking of dependencies and utilities for more reliable and isolated tests in `__tests__/screens/send-destination.spec.tsx`.

**Component and Utility Exports**

* Exported the `phone-input` component from `app/components/phone-input/index.ts` to make it available for use elsewhere in the codebase.

**General Test Code Quality**

* Refactored usages of the `parseDestination` mock to use a strongly typed and consistent mock reference, improving maintainability and clarity in the test suite. [[1]](diffhunk://#diff-165d18520a189995468e21a4e32417a1477a753e09d8a10cad6e73052a23eca5R120-R122) [[2]](diffhunk://#diff-165d18520a189995468e21a4e32417a1477a753e09d8a10cad6e73052a23eca5L171-R543) [[3]](diffhunk://#diff-165d18520a189995468e21a4e32417a1477a753e09d8a10cad6e73052a23eca5L244-R616) [[4]](diffhunk://#diff-165d18520a189995468e21a4e32417a1477a753e09d8a10cad6e73052a23eca5L279-R651)

### CODE REVIEW

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 1]
```diff
@@ -438,27 +569,64 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
...
+          payload: { unparsedDestination: international || displayHandle },
+        })
+        initiateGoToNextScreen(international || displayHandle)
 
-  const inputContainerStyle = React.useMemo(() => {
+        setRawPhoneNumber(international || displayHandle)
+
+        setTimeout(() => {
+          setKeepCountryCode(true)
+        }, 100)
```
[the remark this answers is not in the document]
Comment (JMoises-XCode) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:624]: The setTimeout with 100ms delay for keepCountryCode is intentionally benign and does not pose a race condition risk. 

Using React's state batching or callbacks would add unnecessary complexity for what is essentially a "fire-and-forget" UI polish feature. The current implementation is simple, readable, and fit-for-purpose.

If this were controlling validation logic, API calls, or navigation—then yes, a more robust approach would be needed. But for a UI smoothing timeout, this pattern is appropriate.

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 2]
```diff
@@ -128,6 +166,15 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
     sendBitcoinDestinationReducer,
     defaultDestinationState,
   )
+
+  const activeInputRef = useRef<TInputType>("search")
+
+  const [rawPhoneNumber, setRawPhoneNumber] = useState<string>("null")
```
[the remark this answers is not in the document]
Comment (JMoises-XCode) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx]: Done

File: app/components/phone-input/phone-input.tsx [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+        onSelect={handleCountrySelect}
+        onClose={handleCountryPickerClose}
+        renderFlagButton={({ countryCode, onOpen }) => {
+          return (
+            countryCode && (
+              <TouchableOpacity
+                style={[
+                  styles.countryPickerButtonStyle,
+                  isDisabled && styles.disabledInput,
+                  countryPickerButtonStyle && countryPickerButtonStyle,
```
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/components/phone-input/phone-input.tsx]: Done @grimen!
```suggestion
                  countryPickerButtonStyle,
```

File: app/components/phone-input/phone-input.tsx [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+      />
+      <Input
+        {...testProps("telephoneNumber")}
+        ref={phoneInputRef}
+        placeholder={PLACEHOLDER_PHONE_NUMBER}
+        containerStyle={styles.inputComponentContainerStyle}
+        inputContainerStyle={[
+          styles.inputContainerStyle,
+          isDisabled && styles.disabledInput,
+          inputContainerStyle && inputContainerStyle,
```
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/components/phone-input/phone-input.tsx]: Done! @grimen 
```suggestion
          inputContainerStyle,
```

File: app/screens/send-bitcoin-screen/send-bitcoin-reducer.ts [hunk 5]
```diff
@@ -99,7 +111,7 @@ export const sendBitcoinDestinationReducer = (
         destinationState: DestinationState.Validating,
       }
     case SendBitcoinActions.SetValid:
-      return state.unparsedDestination === action.payload?.unparsedDestination
+      return state.unparsedDestination === action.payload?.unparsedDestination //
```
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-reducer.ts]: Done! @grimen
```suggestion
      return state.unparsedDestination === action.payload?.unparsedDestination
```

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 6]
```diff
@@ -574,12 +797,14 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
...
               destinationState.unparsedDestination
                 ? LL.common.next()
-                : LL.SendBitcoinScreen.destinationIsRequired()
+                : LL.SendBitcoinScreen.destinationRequired()
             }
             loading={destinationState.destinationState === DestinationState.Validating}
             disabled={
               destinationState.destinationState === DestinationState.Invalid ||
-              !destinationState.unparsedDestination
+              destinationState.destinationState === DestinationState.PhoneInvalid ||
```
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:819]: Done! @grimen
```suggestion
              destinationState.destinationState === DestinationState.PhoneInvalid ||
              destinationState.destinationState === DestinationState.PhoneNotAllowed ||
```

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 7]
```diff
@@ -262,13 +294,54 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
...
+
+      if (activeInputRef.current === "search") {
+        if (isValidPhone?.isValid() || isInt(rawInput)) {
+          dispatchDestinationStateAction({
+            type: SendBitcoinActions.SetPhoneNotAllowed,
+            payload: {},
+          })
+          return
+        }
+      }
```
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:322]: Done! This was resolved with the test coverage added in commit https://github.com/blinkbitcoin/blink-mobile/pull/3539/commits/f4c68ddfcbdd52a8304499fdae4c22717de39793

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 8]
```diff
@@ -22,6 +28,7 @@ import { logParseDestinationResult } from "@app/utils/analytics"
 import { toastShow } from "@app/utils/toast"
 import { PaymentType } from "@blinkbitcoin/blink-client"
 import Clipboard from "@react-native-clipboard/clipboard"
+import { CountryCode, parsePhoneNumber, isValidPhoneNumber } from "libphonenumber-js"
 import crashlytics from "@react-native-firebase/crashlytics"
```
[the remark this answers is not in the document]
Comment (grimen) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx]: @esaugomez31 Please have a look if this makes sense practically.

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 9]
```diff
@@ -480,6 +681,8 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
           return styles.validInputContainer
         }
         return styles.warningInputContainer
+      case DestinationState.PhoneInvalid:
+        return styles.errorInputContainer
```
[the remark this answers is not in the document]
Comment (grimen) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:661]: Please have a look if this makes sense practically.

File: ios/fastlane/Fastfile
Comment (grimen) [inline ios/fastlane/Fastfile:1]: Is this intentional? I recall this was fixed in `main` to ensure IOS Appstore approval.

File: app/screens/send-bitcoin-screen/payment-destination/lnurl.ts [hunk 10]
```diff
@@ -126,6 +127,15 @@ const tryGetIntraLedgerDestinationFromLnurl = ({
   return undefined
 }
 
+const isPhoneNumber = (phoneNumber: string): boolean => {
```
Comment (grimen) [inline app/screens/send-bitcoin-screen/payment-destination/lnurl.ts]: This makes sense in another utility library focused on "phone" since such common area and makes sense to be separated from Bitcoin/LN world utility wise. In other words unless one already exist: `utils/phone` + corresponding tests `utils/phone.spec.ts` that implements phone logic that is reusable/common.

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx
Comment (grimen) [inline app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:1]: My general feeling and feedback after reading this code is "why is there not a phone input component with callbacks that can be used in more than once place that incapsulates common phone validation/masking logic"?

**Review by grimen — CHANGES_REQUESTED** [review]
I feel this PR needs a verbal review, my rough opinion is that it can be much simplified with isolated component(s) and utils, but let's discuss on today sync call.

File: app/screens/send-bitcoin-screen/payment-destination/lnurl.ts (hunk 10 above)
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/payment-destination/lnurl.ts]: Done! Created `utils/phone.ts` with `isPhoneNumber` (and also `parseValidPhoneNumber` for reusability). Added corresponding tests in `__tests__/utils/phone.spec.ts`. The function is now imported from the util in both `lnurl.ts` and `send-bitcoin-destination-screen.tsx`

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:1]: The `PhoneInput` component already encapsulates the phone input UI with country picker, formatting (AsYouType), and callbacks. Additionally, the validation logic has been extracted to `utils/phone.ts` with `isPhoneNumber` and `parseValidPhoneNumber` functions, making them reusable across the codebase

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx (hunk 8 above)
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx]: This no longer applies. The phone validation logic was moved to `app/utils/phone.ts` which imports from `libphonenumber-js/mobile`

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx (hunk 9 above)
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:661]: Added. Without this, the error message appears but the input field doesn't show the red border, making the UX inconsistent with other error states

File: ios/fastlane/Fastfile
Comment (esaugomez31) [reply ios/fastlane/Fastfile:1]: Not intentional. This was caused by a merge conflict. Already reverted

File: __tests__/utils/phone.spec.ts [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+    expect(isPhoneNumber("123")).toBe(false)
+    expect(isPhoneNumber("")).toBe(false)
+  })
+
+  it("returns false for usernames that look like numbers", () => {
+    expect(isPhoneNumber("user123")).toBe(false)
+    expect(isPhoneNumber("test@blink.sv")).toBe(false)
+  })
+
+  it("returns true for valid phone numbers without plus sign when format is recognized", () => {
```
[the remark this answers is not in the document]
Comment (grimen) [reply __tests__/utils/phone.spec.ts]: @esaugomez31 Please verify.

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 12]
```diff
@@ -591,22 +829,229 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
...
+      return
+
+    const { rawPhoneNumber } = defaultPhoneInputInfo
+    const rawInput = `+${defaultPhoneInputInfo?.countryCallingCode}${rawPhoneNumber}`
+
+    handleChangeText(rawInput)
+    updateMatchingContacts(rawPhoneNumber)
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [defaultPhoneInputInfo, handleChangeText, updateMatchingContacts])
```
[the remark this answers is not in the document]
Comment (grimen) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:947]: @esaugomez31 Please verify.

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 13]
```diff
@@ -591,22 +829,229 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
...
+            activeInputRef.current === InputType.Phone && inputContainerStyle
+          }
+          bgColor={colors.grey6}
+          keepCountryCode={keepCountryCode}
+        />
+      </View>
+      {activeInputRef.current === InputType.Phone ? (
+        <DestinationInformation destinationState={destinationState} />
+      ) : (
+        <View style={styles.spacerStyle}></View>
```
[the remark this answers is not in the document]
Comment (grimen) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx]: @esaugomez31 Please verify.

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 14]
```diff
@@ -591,22 +829,229 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
...
+    rawPhoneNumber,
+    defaultPhoneInputInfo,
+    destinationState.destinationState,
+    parseValidPhone,
+  ])
+
+  return (
+    <>
+      <View style={styles.textSeparator}>
+        <View style={styles.line}></View>
```
[the remark this answers is not in the document]
Comment (grimen) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx]: @esaugomez31 Please verify.

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 15]
```diff
@@ -591,22 +829,229 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
...
+        />
+      </View>
+      {activeInputRef.current === InputType.Phone ? (
+        <DestinationInformation destinationState={destinationState} />
+      ) : (
+        <View style={styles.spacerStyle}></View>
+      )}
+      {matchingContacts.length > 0 && (
+        <View style={[styles.textSeparator, styles.lastInfoTextStyle]}>
+          <View style={styles.line}></View>
```
[the remark this answers is not in the document]
Comment (grimen) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx]: @esaugomez31 Please verify.

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 16]
```diff
@@ -262,18 +281,51 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
...
+      }
+
+      if (activeInputRef.current === InputType.Search) {
+        if (isValidPhone?.isValid() || isInt(rawInput)) {
+          dispatchDestinationStateAction({
+            type: SendBitcoinActions.SetPhoneNotAllowed,
+            payload: {},
+          })
+          return
+        }
```
[the remark this answers is not in the document]
Comment (grimen) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:321]: @esaugomez31 Please verify.

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 17]
```diff
@@ -447,27 +585,64 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
...
+      initiateGoToNextScreen(displayHandle)
+    },
+    [
+      destinationState.destinationState,
+      lnAddressHostname,
+      parseValidPhone,
+      onFocusedInput,
+      updateMatchingContacts,
+      handleSelection,
+      initiateGoToNextScreen,
```
[the remark this answers is not in the document]
Comment (grimen) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:641]: @esaugomez31 Please verify.

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 18]
```diff
@@ -518,53 +704,102 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
...
+                </Text>
+              </View>
+            </TouchableOpacity>
+          )}
         </View>
-        <DestinationInformation destinationState={destinationState} />
+        {activeInputRef.current === InputType.Search ? (
+          <DestinationInformation destinationState={destinationState} />
+        ) : (
+          <View style={styles.spacerStyle}></View>
```
[the remark this answers is not in the document]
Comment (grimen) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx]: @esaugomez31 Please verify.

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 19]
```diff
@@ -591,22 +829,229 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
...
+        <View style={styles.textInformationWrapper}>
+          <Text style={styles.textInformation}>{LL.SendBitcoinScreen.orBySMS()}</Text>
+        </View>
+      </View>
+      <View
+        onStartShouldSetResponder={() => activeInputRef.current !== InputType.Phone}
+        onResponderRelease={() => onFocusedInput(InputType.Phone)}
+      >
+        <PhoneInput
+          key={1}
```
[the remark this answers is not in the document]
Comment (grimen) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx]: @esaugomez31 Please verify.

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 20]
```diff
@@ -591,22 +829,229 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
...
+      })
+    }
+  }, [
+    destinationState.destinationState,
+    onFocusedInput,
+    parseValidPhone,
+    updateMatchingContacts,
+    willInitiateValidation,
+    waitAndValidateDestination,
+    LL,
```
[the remark this answers is not in the document]
Comment (grimen) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:926]: @esaugomez31 Please verify.

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx [hunk 21]
```diff
@@ -591,22 +829,229 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
...
+          "",
+        )
+        setRawPhoneNumber(phoneNumberWithoutArea)
+      }
+    }
+  }, [
+    rawPhoneNumber,
+    defaultPhoneInputInfo,
+    destinationState.destinationState,
+    parseValidPhone,
```
[the remark this answers is not in the document]
Comment (grimen) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:974]: @esaugomez31 Please verify.

File: __tests__/utils/phone.spec.ts (hunk 11 above)
[the remark this answers is not in the document]
Comment (esaugomez31) [reply __tests__/utils/phone.spec.ts]: Done!

reviewed, Copilot is right, the test name was incorrect. I updated it to match the expected behavior (false without the + prefix).

```diff
diff --git a/__tests__/utils/phone.spec.ts b/__tests__/utils/phone.spec.ts
index d91b535e5..cfe0f15dd 100644
--- a/__tests__/utils/phone.spec.ts
+++ b/__tests__/utils/phone.spec.ts
@@ -44,7 +44,7 @@ describe("isPhoneNumber", () => {
     expect(isPhoneNumber("test@blink.sv")).toBe(false)
   })
 
-  it("returns true for valid phone numbers without plus sign when format is recognized", () => {
+  it("returns false for valid phone numbers without plus sign even when format is recognized", () => {
     expect(isPhoneNumber("14155552671")).toBe(false)
   })
 })
```

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx (hunk 12 above)
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:947]: The `eslint-disable` is intentional. This effect should only run when the phone input changes, not when the validation state changes. If we add `destinationState.destinationState` as a dependency, when the user enters an invalid phone number and the state changes to `PhoneInvalid`, the effect re-runs and calls `handleChangeText()`, which resets the state back to `Entering`. This causes the error message to disappear immediately (in milliseconds). The effect reads `destinationState` for early returns, but it shouldn't react to its changes — this is a valid use case for suppressing the exhaustive-deps rule.

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx (hunk 13 above)
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx]: Done.

```diff
-          <View style={styles.spacerStyle}></View>
+          <View style={styles.spacerStyle} />
```

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx (hunk 14 above)
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx]: Done.

```diff
-        <View style={styles.line}></View>
+        <View style={styles.line} />
```

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx (hunk 15 above)
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx]: Done!

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx (hunk 16 above)
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:321]: This is intentional. The purpose of this check is UX guidance: when the user types only digits in the search field, we assume they're trying to enter a phone number and show an error directing them to use the phone input instead. While technically a numeric username like "12345" would be blocked

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx (hunk 17 above)
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:641]: Done! Followed the suggestion. This doesn't change the behavior since setState functions are guaranteed stable by React, but added them for consistency with the pattern.

```diff
       initiateGoToNextScreen,
+      setKeepCountryCode,
+      setRawPhoneNumber,
     ],
```

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx (hunk 18 above)
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx]: Done!

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx (hunk 19 above)
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx]: Done. Removed it since there's only one instance of this component and it's not inside a loop

```diff
         <PhoneInput
-          key={1}
           rightIcon={
```

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx (hunk 20 above)
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:926]: Done! Added for consistency, though setState functions are stable by React's guarantee

```diff
     LL,
+    setRawPhoneNumber,
   ])
```

File: app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx (hunk 21 above)
[the remark this answers is not in the document]
Comment (esaugomez31) [reply app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx:974]: Done! Similar to the previous case - setState functions are stable, but added for consistency with React Hook best practices
